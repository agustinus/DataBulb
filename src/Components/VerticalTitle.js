import React from 'react';
import {StyleSheet, Text} from 'react-native';

export default class VerticalTitle extends React.Component {
  render() {
    return <Text style={styles.title}>MOBILE DATA USAGE</Text>;
  }
}

const styles = StyleSheet.create({
  title: {
    fontWeight: 'bold',
    fontSize: 20,
    textAlign: 'right',
    transform: [{rotate: '-90deg'}],
    color: '#373737',
    width: 250,
    marginTop: 116,
    letterSpacing: 1.67,
    textShadowOffset: {
      width: 0,
      height: 1,
    },
    textShadowColor: '#0000006A',
    textShadowRadius: 2,
  },
  loadingContainer: {
    marginTop: 80,
    position: 'absolute',
    left: 0,
    right: 0,
    top: 0,
    bottom: 0,
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: '#F5FCFF88',
  },
});
