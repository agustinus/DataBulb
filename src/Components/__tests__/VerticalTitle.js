import React from 'react';
import VerticalTitle from '../VerticalTitle';

import renderer from 'react-test-renderer';

describe('VerticalTitle', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<VerticalTitle />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
