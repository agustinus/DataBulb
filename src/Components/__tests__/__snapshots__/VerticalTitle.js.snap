// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerticalTitle renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#373737",
      "fontSize": 20,
      "fontWeight": "bold",
      "letterSpacing": 1.67,
      "marginTop": 116,
      "textAlign": "right",
      "textShadowColor": "#0000006A",
      "textShadowOffset": Object {
        "height": 1,
        "width": 0,
      },
      "textShadowRadius": 2,
      "transform": Array [
        Object {
          "rotate": "-90deg",
        },
      ],
      "width": 250,
    }
  }
>
  MOBILE DATA USAGE
</Text>
`;
